### PR TITLE
feat: use `console.time` for timing

### LINKED ISSUE #2406: Configurator does not load hex files
since latest build the configurator does not load any hex files.

Screenshot_console

imo it comes from #2368 ... @chmelevskij pls can you fix this

### PR DESCRIPTION
**Pull request by chmelevskij**
Fixes #2406 

@McGiverGim used `console.time` stuff. 

@TheIsotopes could you verify it works?

### CODE REVIEW

**Comment by TheIsotopes** [comment]
yes, works again ... thx 👍

**Review by McGiverGim — CHANGES_REQUESTED** [review]
The microtime function is only used twice. One where you have fixed with this, the other here:

https://github.com/betaflight/betaflight-configurator/blob/4be400a5c6c0e9f1e45d90ae6c3ba0d4a142c81d/src/js/ConfigInserter.js#L89-L105

I think is better to remove the function at all and fix the problem in both places.

File: src/js/workers/hex_parser.js [hunk 1]
```diff
@@ -93,13 +93,14 @@ function read_hex_file(data) {
     }
 }
 
+const TIME_LABEL = 'HEX_PARSER - File parsed in';
+
 onmessage = function(event) {
-    const timeParsingStart = microtime(); // track time
+    console.time(TIME_LABEL)
```
Comment (McGiverGim) [inline src/js/workers/hex_parser.js:99]: Sonar issue: missing semicolon.

Comment (chmelevskij) [reply src/js/workers/hex_parser.js:99]: Lol, this will go into eslint for sure 😆

**Comment by limonspb** [comment]
fixes the issue for me too

File: src/js/workers/hex_parser.js (hunk 1 above)
Comment (mikeller) [reply src/js/workers/hex_parser.js:99]: Are you going to fix it though? :wink:

**Review by mikeller — APPROVED** [review]

**Review by McGiverGim — APPROVED** [review]

**Comment by Asizon** [comment]
Here is some wrong:
image

**Comment by TheIsotopes** [comment]
@Asizon #2414 will fix this

**Pull request merged**